.travis.yml

```diff
@@ -1,5 +1,5 @@
 sudo: required
-dist: xenial
+dist: disco
```

Comment:
The problem is that ubuntu18.04 only supports nsis2.5 and doesn't support the attribute ManifestDPIAware. But we could make the compromise that we use ubuntu20.04. It is a LTS-version und should release on April 23, 2020.

Reply:
Then we could also remove the ``before_install`` job at the ```.travis.yml```file.

Reply:
Well this is just Travis and we do run openvpn-build on every commit in Buildbot. So I would be ok to switching to 20.04 for Travis in preparation for migrating the build system to 20.04 as well. 

@chipitsine opinions on this?

Reply:
Oh, one thing. You mentioned that ManifestDPIAware is not present in nsis2.5. We use that for building official packages. Does NSIS 2.5 just warn if that parameter is not available?

Reply:
...instead of falling on its face :smile:

Reply:
Sadly it will stop :/

Why don't you want build your official packages with nsis3? I think this won't change anything.

Reply:
I could help if there would be any problems exist.

Reply:
in theory, does NSIS supports conditional ? like "if 3.0, then use those directives"

Reply:
as for ubuntu 19.10, I think it makes sense if we will build for both 18.04 and 19.10
if 19.10 included newer mingw compilers, so we can run build and see if there are issues on new compilers.

as alternative, I thought about using Github Actions and running Fedora Rawhide (to get the newest possible compilers)

Reply:
@chipitsine yes nsis could this (``!if 0n1 > 0 ; >= 3.0b0``).

Reply:
Github Actions sounds good.

Reply:
> Sadly it will stop :/
> 
> Why don't you want build your official packages with nsis3? I think this won't change anything.

Simply because our build rig in [openvpn-vagrant](https://github.com/OpenVPN/openvpn-vagrant/)  _is_ Ubuntu 18.04-based. We can upgrade that to to 20.04 for sure, but then we also need to make sure that the "official" [fork of osslsigncode](https://github.com/mtrojnar/osslsigncode) works as properly. We do have some other NSIS patches in the queue, and doing the NSIS 3.0 migration would make sense in this context I think. All in one go